src/app/commands/cmd_import_sprite_sheet.cpp

```diff
@@ -144,14 +152,36 @@ class ImportSpriteSheetWindow : public app::gen::ImportSpriteSheet
       std::max<int>(1, h));
   }
 
+  gfx::Size getPaddingFromEntries() {
+  int padW = horizontalPadding()->textInt();
```

Comment:
```suggestion
    int padW = horizontalPadding()->textInt();
```
Minor change, the indentation needs a couple of white spaces.